src/Order.php

```diff
@@ -2,13 +2,19 @@
 
 namespace Razorpay\Api;
 
+use Requests;
+
 class Order extends Entity
 {
     /**
      * @param $id Order id description
      */
     public function create($attributes = array())
     {
+        $attributes = json_encode($attributes);
```

Comment:
@ankitdas13 , Is there any tests needs to be updated for this?

Reply:
@ramth05 we dont have to update any where for this

Reply:
@ramth05 added negative test cases for create order